tests/cli/test_cli_regression_mol.py

```diff
@@ -404,7 +419,7 @@ def test_hyperopt_quick(monkeypatch, data_path, tmp_path):
         "-i",
         input_path,
         "--epochs",
-        "1",
+        "6",
```

Comment:
Can you also change the number of epochs from 1 to 6 in the Optuna test and add the "--search-parameter-keywords", "all", in the optuna test to make them compatible?